### DISCUSSION TITLE
Empty 定义

### DISCUSSION THREAD

**Discussion by Just-focus** [jdf2e/nutui-react#184] [Q&A] [answered]
请问这些值是随机生成的吗

<img width="598" alt="image" src="https://user-images.githubusercontent.com/18374969/181670791-8227a14c-7ea6-42e0-90e2-87d153331dc4.png">

**Comment by oasis-cloud** [comment, marked as answer]
可以查看 https://github.com/jdf2e/nutui-react/blob/main/scripts/extract-language.js
